recipes/libpq/all/conanfile.py

```diff
@@ -189,6 +199,7 @@ def _remove_unused_libraries_from_package(self):
         else:
             globs = [
                 os.path.join(self.package_folder, "lib", "libpq.so*"),
+                os.path.join(self.package_folder, "lib", "*.dll"),
                 os.path.join(self.package_folder, "bin", "*.dll"),
                 os.path.join(self.package_folder, "lib", "libpq*.dylib")
             ]
```

Comment:
I think cleanup is incomplete.
I propose to replace all the content of `_remove_unused_libraries_from_package()` by:

```python

    def _remove_unused_libraries_from_package(self):
        if self.settings.os == "Windows":
            rm(self, "*.dll", os.path.join(self.package_folder, "lib"))
            if self.options.shared:
                for lib in ("libpq.a", "libpgcommon.a", "libpgcommon_shlib.a", "libpgport.a", "libpgport_shlib.a"):
                    rm(self, lib, os.path.join(self.package_folder, "lib"))
            else:
                rm(self, "*.dll", os.path.join(self.package_folder, "bin"))
                rm(self, "*.dll.a", os.path.join(self.package_folder, "lib"))
        else:
            if self.options.shared:
                rm(self, "*.a", os.path.join(self.package_folder, "lib"))
            else:
                rm(self, "*.so*", os.path.join(self.package_folder, "lib"))
                rm(self, "*.dylib", os.path.join(self.package_folder, "lib"))

```

Reply:
I'll update my PR with changes from yours.